### DISCUSSION TITLE
Subdomainradar.io

### DISCUSSION THREAD

**Discussion by Sh4d0wHunt3rX** [blacklanternsecurity/bbot#1804] [Module Requests]
Columbus site is not returning any results for more than a week and returns 502 Bad Gateway. I appreciate if you can add a module for Subdomainradar.io which is very powerful.

https://subdomainradar.io/
https://api.subdomainradar.io/docs

Thanks 🙏

**Comment by TheTechromancer** [comment]
@AlexandreVandamme59 nice job on this API; we're working on integrating it into BBOT.

I have a question about groups/enumerators. What's the significance of those? is it important how we choose them?

**Reply by AlexandreVandamme59** [reply]
Thank you for your interest and the question! I’ve updated our FAQ to address this, but let me provide a clear answer here as well.

On **SubdomainRadar.io**, enumerators are the sources or techniques used to discover subdomains. The **groups** (Fast, Medium, Deep) are predefined combinations of these enumerators, tailored for different use cases:

- **Fast** (Free & Paid Plan): Optimized for speed, offering quick results with moderate coverage. (5 to 40 seconds)
- **Medium** (Paid Plan): A balanced approach, combining speed and depth. (20 to 80 seconds)
- **Deep** (Paid Plan): Uses the full set of enumerators, including brute-force techniques, paid APIs (30+), and paid data sources, for comprehensive subdomain discovery. (30 to 160 seconds)

While it is technically possible to select individual enumerators manually, **we strongly recommend using the pre-built groups**. These groups are regularly curated and updated to ensure optimal subdomain enumeration. Manually selecting enumerators can lead to suboptimal results, so it's best to rely on the optimized groups we’ve designed.

I hope this clarifies things for your integration with BBOT. Feel free to reach out if you have any further questions!

You can also find more details in our FAQ, which has been updated here: [https://subdomainradar.io/pricing](https://subdomainradar.io/pricing).

**Reply by TheTechromancer** [reply]
Thanks @AlexandreVandamme59, that makes sense.

**Comment by TheTechromancer** [comment]
Subdomainradar module has been merged into dev 👍

@Sh4d0wHunt3rX @AlexandreVandamme59
